test/integration/file-utils.spec.js

```diff
@@ -1,152 +0,0 @@
-'use strict';
-
-var utils = require('../../lib/utils');
-var fs = require('fs');
-var path = require('path');
-var os = require('os');
-var mkdirp = require('mkdirp');
-var rimraf = require('rimraf');
-
-describe('file utils', function() {
-  var tmpDir = path.join(os.tmpdir(), 'mocha-file-lookup');
-  var existsSync = fs.existsSync;
-  var tmpFile = path.join.bind(path, tmpDir);
-  var symlinkSupported = process.platform !== 'win32';
-
-  beforeEach(function() {
-    this.timeout(2000);
-    makeTempDir();
-
-    fs.writeFileSync(tmpFile('mocha-utils.js'), 'yippy skippy ying yang yow');
-    if (symlinkSupported) {
-      fs.symlinkSync(tmpFile('mocha-utils.js'), tmpFile('mocha-utils-link.js'));
-    }
-  });
-
-  describe('.lookupFiles', function() {
```

Comment:
This function still exists, please don't delete the corresponding tests.

Reply:
Oops. I restored the test cases